pkg/kubectl/cmd/resource/get.go

```diff
@@ -181,6 +182,10 @@ func (options *GetOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args
 		return nil
 	}
 
+	if len(args) > 0 && args[0] == "all" {
```

Comment:
There's no guarantee that's where "all"  will be coming in the list of args, users could do e.g. `kubectl get nodes,all`. Also, we want to remove the references to hardcoded "all" on the client, it's now a server-side concept that the client doesn't have detailed knowledge about.

Reply:
Hmm... that's fair, but how else are you proposing to go with this?

Reply:
Nvmd, just noticed the next comment ;)